crispy_forms/tests/test_layout_objects.py

```diff
@@ -575,3 +576,28 @@ def test_non_ascii_chars_in_container_name(self):
         name = "テスト"
         test_container = Container(name, "val1", "val2")
         assert test_container.css_id == name
+
+    @only_bootstrap3
```

Comment:
Sounds good. I'm running into a problem where `parse_form()` appears to be incorrectly parsing bootstrap4 content. In the bootstrap4 tests, I'm using the same forms as in the bootstrap3 tests. When I print the output of `parse_form()`, I get the following (notice the red circle indicating a class attribute with no actual classes listed):
<img width="1172" alt="Screen Shot 2022-01-20 at 6 24 21 PM" src="https://user-images.githubusercontent.com/37155082/150438657-3f59f6e8-78b4-4a17-9530-7457e06d5086.png">

The form used in that screen shot was `SampleForm()` and here's the code used in the test:
```
form = SampleForm()
form.helper = FormHelper()
form.helper.layout = Layout(Modal(Field("first_name")))
 ```
Funny thing is, I can't just change the result file to match that output because then `parse_expected()` complains that it was expecting a string following the `class` attribute. Is it possible that the `parse_form()` function has an error?

Reply:
My fix for this is in #1208. I think then there may be a couple of tricky space issues with Django 2.2 (Is it April yet?)